src/rules/require-top-level-describe.ts

```diff
@@ -0,0 +1,44 @@
+import { TSESTree } from '@typescript-eslint/experimental-utils';
+
+import { createRule, isDescribe, isHook, isTestCase } from './utils';
+
+export default createRule({
+  name: __filename,
+  meta: {
+    docs: {
+      category: 'Best Practices',
+      description:
+        'Prevents test cases and hooks to be outside of a describe block',
+      recommended: false,
+    },
+    messages: {
+      unexpectedMethod: 'All test cases must be wrapped in a describe block.',
```

Comment:
should we have separate test case and hook error message?

Reply:
We could go with `'All test cases and hooks must be wrapped in a describe block.'` but it feels long to me.
Having separate error messages make sense to me. I'll add a test in that case!